src/Npgsql/Replication/PgOutput/Messages/PreparedTransactionMessageBase.cs

```diff
@@ -0,0 +1,37 @@
+﻿using NpgsqlTypes;
+using System;
+
+namespace Npgsql.Replication.PgOutput.Messages;
+
+/// <summary>
+/// Abstract base class for Logical Replication Protocol prepare and begin prepare message
+/// </summary>
+public abstract class PreparedTransactionMessageBase : TransactionControlMessage
```

Comment:
Maybe call this PreparedTransactionControlMessage for consistency with TransactionControlMessage?